lib/space.rb

```diff
@@ -1,22 +1,23 @@
 require 'pg'
 
 class Space
-  attr_reader :id, :name
+  attr_reader :id, :name, :description
 
-  def initialize(id:, name:)
+  def initialize(id:, name:, description:)
     @id = id
     @name = name
+    @description = description
   end
 
-  def self.create(name:)
+  def self.create(name:, description: '')
```

Comment:
should description be empty by default, or required, not sure? :thinking:

Reply:
Fair point. We've changed the implementation to remove the default argument value, as we felt that requiring description to be explicitly set each time create is called led to easier to read code and fewer mistakes.